filter_plugins/babylon.py

```diff
@@ -27,6 +29,25 @@ def filter_job_vars_secrets_to_dynamic_job_vars(anarchy_governor):
             dynamic_job_var_secrets.append(dynamic_job_var_secret)
     return dynamic_job_var_secrets
 
+def __insert_unvault_string(value, vaulted_values):
+    if isinstance(value, dict):
+        return { k: __insert_unvault_string(v, vaulted_values) for k, v in value.items() }
+    elif isinstance(value, list):
+        return [ __insert_unvault_string(v, vaulted_values) for v in value ]
+    if isinstance(value, str) and value.startswith('$ANSIBLE_VAULT;'):
```

Comment:
I think we want to support strings that are built like the following:

```yaml
the_dev_secret: !vault |
          $ANSIBLE_VAULT;1.2;AES256;dev
          30613233633461343837653833666333643061636561303338373661313838333565653635353162
          3263363434623733343538653462613064333634333464660a663633623939393439316636633863
          61636237636537333938306331383339353265363239643939666639386530626330633337633833
          6664656334373166630a363736393262666465663432613932613036303963343263623137386239
          6330
```

( taken from the official documentation)

If you change your test or add another one with:

```yaml
      test_vaulted_string: |
        $ANSIBLE_VAULT;1.1;AES256
        62636665646636306161303836626134616161633236633065653737326538663931613462366666
        6634313965303664623034666461353933333564623737350a666536323264353031653765313232
        63633437346264616339623734353966393238346365323766333464623564646161386335356431
        6166306331306130350a386634356432653935313931303432623333386134336339633032613235
        6431
```

it fails with:

```
TASK [Check insert_unvault_string processing] *************************************************************************************************************************************************
[WARNING]: Skipping plugin (/home/fridim/sync/dev/vault-pr/filter_plugins/test_babylon.py) as it seems to be invalid: No module named 'babylon'
fatal: [localhost]: FAILED! => {
    "assertion": "vars.processed_test_vars[test_vaulted_list_1_vaulted_value_var] == test_vars.test_vaulted_string",
    "changed": false,
    "evaluated_to": false,
    "msg": "Assertion failed"
}
```

(Works with `>-`  instead of `|`. )